### PR TITLE
fix: guard negative usage log latency

### PR DESCRIPTION
**Pull request by MisonL**
## Summary
- guard usage log use_time against negative elapsed time across quota and error-log paths
- only persist frt when first-response latency is valid, and hide historical negative values in the log table
- restore Claude file-content conversion and preserve pre-initialized stream status during stream scanning

## Verification
- go test ./... -count=1
- cd web && bunx eslint src/components/table/usage-logs/UsageLogsColumnDefs.jsx src/hooks/usage-logs/useUsageLogsData.jsx --cache
- cd web && bun run build

## Evidence
- independent dev service verified on port 3002 with isolated SQLite
- post-fix logs showed use_time >= 0 and no negative frt entries under real request flow

## Summary by CodeRabbit

* **Bug Fixes**
  * Prevented negative/invalid elapsed-time values from affecting logs, quotas, and usage calculations (now clamped to zero)
  * More reliable first-response timing across streaming handlers to ensure correct latency reporting
  * Stream handling preserves pre-existing stream state and enforces a safe minimum streaming timeout
  * UI now hides invalid or negative time values to avoid misleading displays

* **Tests**
  * Added unit tests covering elapsed-time clamping, first-response latency edge cases, and related log/usage outputs

### CODE REVIEW

File: relay/channel/claude/relay-claude.go [hunk 1]
```diff
@@ -431,15 +431,14 @@ func RequestOpenAI2ClaudeMessage(c *gin.Context, textRequest dto.GeneralOpenAIRe
...
-					// FIXME
-					//case dto.ContentTypeFile:
-					//	claudeFileMessage, err := buildClaudeFileMessage(c, mediaMessage.GetFile())
-					//	if err != nil {
-					//		return nil, err
-					//	}
-					//	if claudeFileMessage != nil {
-					//		claudeMediaMessages = append(claudeMediaMessages, *claudeFileMessage)
-					//	}
+					case dto.ContentTypeFile:
```
Comment (seefs001) [inline relay/channel/claude/relay-claude.go]: ？这是干啥

Comment (MisonL) [reply relay/channel/claude/relay-claude.go]: 这段确实和本 PR 目标无关，是我之前收范围时误带进来的 Claude file handling 改动。现在已经从 #4057 剥离并推送，只保留负数延迟修复相关变更。

**Comment by MisonL** [comment]
已在 2026-04-08 将本 PR 分支 rebase 到最新 upstream/main（dc83c4af），并已 force-push 更新当前 head：`ad8f2fd1`。

本次仅处理基线同步与冲突收敛，没有额外扩展需求范围；如本地 review diff 仍显示旧基线，请刷新后按新 head 继续审阅。

**Comment by MisonL** [comment]
Closing this PR for now.

I will continue refining and completing this work independently on my fork branches, and I will not submit further PRs to this repository.

Thank you for the review time so far.
